README.md

```diff
@@ -4,111 +4,43 @@ An implementation of ripple keypairs & wallet generation using
 [elliptic](https://github.com/indutny/elliptic) which supports rfc6979 and
 eddsa deterministic signatures.
 
-## Generate a random wallet
-```js
-> var generateWallet = require('ripple-keypairs').generateWallet;
-> generateWallet({type: 'ed25519'});
-{ seed: 'sEd7t79mzn2dwy3vvpvRmaaLbLhvme6',
-  accountID: 'r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ',
-  publicKey: 'ED5F5AC8B98974A3CA843326D9B88CEBD0560177B973EE0B149F782CFAA06DC66A' }
-```
+## API Methods
 
-## Derive a wallet from a seed
-```js
-> var walletFromSeed = require('ripple-keypairs').walletFromSeed;
-> walletFromSeed('sEd7t79mzn2dwy3vvpvRmaaLbLhvme6');
-{ seed: 'sEd7t79mzn2dwy3vvpvRmaaLbLhvme6',
-  accountID: 'r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ',
-  publicKey: 'ED5F5AC8B98974A3CA843326D9B88CEBD0560177B973EE0B149F782CFAA06DC66A' }')
 ```
-
-## Generate random validator keys
-```js
-> var generateValidatorKeys = require('ripple-keypairs').generateValidatorKeys;
-> generateValidatorKeys();
-{ seed: 'ssC7Y9LMKhuzFMVueaj2fnTuGLftA',
-  publicKey: 'n9MU2RsULUayZnWeLssjbMzVRPeVUUMgiPYTwe8eMgpdGDWp5t8C' }
+generateSeed({entropy?: Array<integer>, algorithm?: string}) -> string
 ```
+Generate a seed that can be used to generate keypairs. Entropy can be provided as an array of 32-bit integers. If provided, it must be at least 16 words long. If not provided, entropy will be automatically generated. The "algorithm" defaults to "secp256k1", but can also be set to "ed25519". The result is a seed encoded in base58, starting with "s".
```

Comment:
JS doesn't have a 32bit word type. And we are here interpreting the array
of 16 js Numbers as 16 octets, not 16 words.